ffi/capi/bindings/cpp/ICU4XCaseMapper.hpp

```diff
@@ -44,27 +44,35 @@ class ICU4XCaseMapper {
    * Returns the full lowercase mapping of the given string
    * 
    * See the [Rust documentation for `lowercase`](https://docs.rs/icu/latest/icu/casemap/struct.CaseMapper.html#method.lowercase) for more information.
+   * 
+   * Warning: Passing ill-formed UTF-8 is undefined behavior (and may be memory-unsafe).
```

Comment:
Question: This is only in Cpp1, right? In Cpp2 this becomes the new safe C++ code path with `diplomat_is_str`?

Reply:
Yes